Detectors/CTP/workflowScalers/src/ctp-ccdb-orbit.cxx

```diff
@@ -0,0 +1,140 @@
+  const std::string prodCCDB = "http://o2-ccdb.internal";
+  bpo::variables_map vm;
+  bpo::options_description opt_general("Usage:\n  " + std::string(argv[0]) +
+                                       " Write orbit staff to ccdb\n");
+  bpo::options_description opt_hidden("");
+  bpo::options_description opt_all;
+  bpo::positional_options_description opt_pos;
+  try {
+    auto add_option = opt_general.add_options();
+    add_option("help,h", "Print this help message");
+    add_option("output-file,o", bpo::value<std::string>()->default_value("none"), "output file name, none - file not created");
+    add_option("output-dir,d", bpo::value<std::string>()->default_value("./"), "output dir");
+    add_option("ccdb", bpo::value<std::string>()->default_value("test"), "choose databse: test or prod else no writing to ccdb");
+    add_option("action,a", bpo::value<std::string>()->default_value(""), "sox - first orbit otherwise orbit reset");
+    add_option("run-number,r", bpo::value<int64_t>()->default_value(123), "run number");
+    add_option("testReset,t", bpo::value<bool>()->default_value(0), "0 = CTP/Calib/OrbitReset; 1 = CTP/Calib/OrbitResetTest");
+    add_option("sox-orbit,x", bpo::value<int64_t>()->default_value(0), "SOX orbit");
+
+    //
+    opt_all.add(opt_general).add(opt_hidden);
+    bpo::store(bpo::command_line_parser(argc, argv).options(opt_all).positional(opt_pos).run(), vm);
+    if (vm.count("help")) {
+      std::cout << opt_general << std::endl;
+      exit(0);
+    }
+    bpo::notify(vm);
+  } catch (bpo::error& e) {
+    std::cerr << "ERROR: " << e.what() << std::endl
+              << std::endl;
+    std::cerr << opt_general << std::endl;
+    exit(1);
+  } catch (std::exception& e) {
+    std::cerr << e.what() << ", application will now exit" << std::endl;
+    exit(2);
+  }
+  std::string action = vm["action"].as<std::string>();
+  std::vector<int64_t> vect;
+  std::string ccdbPath;
+  auto now = std::chrono::system_clock::now();
+  long tt = std::chrono::duration_cast<std::chrono::milliseconds>(now.time_since_epoch()).count();
+  if (action == "sox") {
+    // write to CTP/Calib/FirstRunOrbit
+    std::cout << "===> FirsRunOrbit" << std::endl;
+    vect.push_back(tt);
+    vect.push_back(vm["run-number"].as<int64_t>());
+    vect.push_back(vm["sox-orbit"].as<int64_t>());
+    ccdbPath = "CTP/Calib/FirstRunOrbit";
+  } else {
+    // write to CTP/Calib/OrbitReset
+    std::cout << "===> ResetOrbit" << std::endl;
+    vect.push_back(tt);
+    ccdbPath = "CTP/Calib/OrbitReset";
+    if (vm["testReset"].as<bool>()) {
+      ccdbPath += "Test";
+    }
+  }
+  //
+  std::string ccdbAddress;
+  if (vm["ccdb"].as<std::string>() == "prod") {
+    std::cout << " Writing to " << prodCCDB << std::endl;
+    ccdbAddress = prodCCDB;
+  } else if (vm["ccdb"].as<std::string>() == "test") {
+    std::cout << " Writing to " << testCCDB << std::endl;
+    ccdbAddress = testCCDB;
```

Comment:
Why not using real ccdb url in the option instead of the test/prod alias? We may need to update some other ccdb, e.g. local one.

Reply:
ok, done